kubespawner/objects.py

```diff
@@ -110,13 +113,6 @@ def make_pod_spec(
 
     pod.spec = V1PodSpec()
 
-    security_context = V1PodSecurityContext()
```

Comment:
Do you mean attach a empty V1PodSecurityContext() to pod.spec?

For fs_group and run_as_user in security, it's better to set in container otherwise initContainers will be failed due to no run_as_user found